internal/core/metadata/v2/application/device.go

```diff
@@ -49,3 +50,29 @@ func AddDevice(d models.Device, ctx context.Context, dic *di.Container) (id stri
 
 	return addedDevice.Id, nil
 }
+
+// CheckDeviceExistsById checks the device existence by id
+func CheckDeviceExistsById(id string, dic *di.Container) (exists bool, err errors.EdgeX) {
+	if id == "" {
```

Comment:
Should also check it is a valid UUID? I saw this in other PRs.

Reply:
Yes, added the UUID validation.